src/DataObjects/QueuedJobDescriptor.php

```diff
@@ -126,6 +127,8 @@ class QueuedJobDescriptor extends DataObject
      */
     private static $searchable_fields = [
```

Comment:
Alternatively, we could remove $searchable_fields altogether and rely on the the default. However, you end up with all the summary fields in there. Most of them are date field which won't provide value unless you implement a range filter.

In any case, Status and Type seem like the most useful filter anyway.